webview/src/plots/components/comparisonTable/cell/ComparisonTableMultiCell.tsx

```diff
@@ -30,7 +30,10 @@ export const ComparisonTableMultiCell: React.FC<{
   useEffect(() => {
     window.clearTimeout(changeDebounceTimer.current)
     changeDebounceTimer.current = window.setTimeout(() => {
-      if (currentStep === values?.[plot.id]?.[path]) {
+      const isOnMount =
+        values?.[plot.id]?.[path] === undefined && currentStep === 0
```

Comment:
Saw that our windows tests were sometimes failing with:

<img width="1268" alt="Screenshot 2023-08-15 at 7 57 34 AM" src="https://github.com/iterative/vscode-dvc/assets/43496356/36033bf1-9172-483f-b0d3-67e2ba8f92ee">

which made me realize that we were sending unneeded calls on mount.